lib/course_planner_web/models/summary_helper.ex

```diff
@@ -77,7 +77,12 @@ defmodule CoursePlanner.SummaryHelper do
 
          Timex.compare(class_starting_at_datetime, time) >= 0
        end)
-       |> Enum.sort(&(&1.date <= &2.date and &1.starting_at <= &2.starting_at))
+       |> Enum.sort(fn(class1, class2) -> true
```

Comment:
what is this `true` in the end of the line?

Reply:
ohh i am sorry that remains from a test